karma.conf.js

```diff
@@ -150,17 +25,15 @@ module.exports = function (config) {
 
     // list of files / patterns to load in the browser
     // all dependancies should be traced through here
-    files: [ 'test/unit.js' ],
+    files: files,
 
     // list of files to exclude
     exclude: [],
 
     // preprocess matching files before serving them to the browser
```

Comment:
The comment was relating to the specific preprocessors, so it'd make sense to move it with the code block – that info could easily be found by looking at the karma site though